src/Sandbox.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+const Sandbox = () => {
+  const [userInput, setUserInput] = useState('');
+  const [data, setData] = useState(null);
+  const API_URL = 'https://stranger-things-api.herokuapp.com/api/v1/characters'
+
+  const fetchData = () => {
+    return fetch(`${API_URL}/${userInput}`)
+      .then(res => res.json())
+      .then(setData)
+      .catch(console.log);
+  }
+
+  return (
+    <section id="endpoint-sandbox">
+      <div>
+        <label htmlFor="endpoint"><h3>Try out an endpoint:</h3></label>
+        <p id="user-input">
+        <span>/api/v1/characters/
```

Comment:
```suggestion
        <span>/api/v1/characters
```
I think we should remove the slash since some queries don't require it